client/signup/steps/site-picker/site-picker-submit.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { PLAN_FREE } from 'lib/plans/constants';
+import { getSite } from 'state/sites/selectors';
+import SignupActions from 'lib/signup/actions';
+
+class SitePickerSubmit extends Component {
+	componentWillMount() {
+		const {
+				stepSectionName,
+				stepName,
+				goToStep,
+				selectedSite
+			} = this.props,
+			hasPaidPlan = selectedSite && selectedSite.plan && selectedSite.plan.product_slug !== PLAN_FREE;
```

Comment:
I tried. It didn't work.

But this one has grown too big as it is. I also think the plans page is broken, as it's showing free plan for almost everything, so will try and take a look at that later.